scripts/release-notes.txt

```diff
@@ -1,5 +1,15 @@
 Release Notes of the ESBMC model checker
 
+*** Version 6.3
+* Stack property support
+* ESBMC now supports Microsoft extensions
+* -f an -W flags are forwarded into clang
```

Comment:
"ESBMC now supports Microsoft extensions -f an -W flags are forwarded into clang" -> "ESBMC now supports Microsoft extensions -f an -W flags, which are forwarded into clang".

Reply:
They refer to two different PRs